When building `x64-osx`:
```
In file included from /Volumes/data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/ACE_crc_ccitt.cpp:1:
In file included from /Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/ACE.h:886:
In file included from /Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/ACE.inl:2:
In file included from /Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/OS_NS_unistd.h:27:
In file included from /Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/Time_Value.h:24:
In file included from /Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/Truncate.h:24:
/Users/vagrant/Data/buildtrees/ace/src/ACE-src-7-03b4f81306.clean/ace/Global_Macros.h:61:3: error: ACE/TAO require C++11 compliance, please upgrade your compiler and/or fix the platform configuration for your environment
# error ACE/TAO require C++11 compliance, please upgrade your compiler and/or fix the platform configuration for your environment
```

--- a/ports/ace/portfile.cmake
+++ b/ports/ace/portfile.cmake
@@ -104,7 +104,7 @@ elseif(VCPKG_TARGET_IS_LINUX)
 elseif(VCPKG_TARGET_IS_OSX)
   set(SOLUTION_TYPE gnuace)
   file(WRITE ${ACE_SOURCE_PATH}/config.h "#include \"ace/config-macosx.h\"")
-  file(WRITE ${ACE_ROOT}/include/makeinclude/platform_macros.GNU "include $(ACE_ROOT)/include/makeinclude/platform_macosx.GNU")
+  file(WRITE ${ACE_ROOT}/include/makeinclude/platform_macros.GNU "c++11=1\ninclude $(ACE_ROOT)/include/makeinclude/platform_macosx.GNU")
 endif()
 
 if(VCPKG_TARGET_IS_UWP)

--- a/versions/a-/ace.json
+++ b/versions/a-/ace.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "6883a76a9f9f86e05dc97f1a9364cb7f5365ab00",
+      "git-tree": "fefdf115c09a6996c55262dd9a635647dc7d8440",
       "version-string": "7.0.0",
       "port-version": 0
     },

